src/features/tracking-cookies-1p.js

```diff
@@ -116,6 +118,8 @@ function applyCookieExpiryPolicy () {
 
 // Set up 1st party cookie blocker
 export function load (args) {
+    trackerHosts.clear()
```

Comment:
this is not related to globals, but related to testing. This file has trackerHosts variable which was retaining state between tests, there was no way to clear it.